spanner_orm/model.py

```diff
@@ -42,16 +44,22 @@ def __new__(mcs, name, bases, attrs, **kwargs):
 
     table = None
     schema = {}
+    relations = {}
     old_attrs = attrs.copy()
     for key, value in old_attrs.items():
+      if key == '__table__':
+        table = attrs.pop(key)
       if isinstance(value, field.Field):
         value.name = key
         schema[key] = attrs.pop(key)
-      elif key == '__table__':
-        table = attrs.pop(key)
+      elif isinstance(value, relationship.Relationship):
+        value.name = key
+        relations[key] = attrs.pop(key)
```

Comment:
Should we add
```
else:
  raise SomeKindOfError()
```
That way if this somehow gets in a bad state, we'll know about it more easily.

Reply:
this iterates over all the attributes in a class, which includes methods and class constants. we only want to intercept attributes of certain types and process them differently, we don't want to raise errors on attributes not of those types